wrapper-generator/src/main/kotlin/it/krzeminski/githubactions/wrappergenerator/generation/Generation.kt

```diff
@@ -221,6 +221,19 @@ private fun Metadata.linkedMapOfInputs(inputTypings: Map<String, Typing>): CodeB
     }
 }
 
+private fun TypeSpec.Builder.addMaybeDeprecated(coords: ActionCoords): TypeSpec.Builder {
+    if (coords.deprecatedByVersion == null) {
+        return this
+    }
+    val newerClass = coords.copy(version = coords.deprecatedByVersion)
+    addAnnotation(
+        AnnotationSpec.builder(Deprecated::class)
+            .addMember("message = %S", "This action has a newer major version")
```

Comment:
Could we mention the new version here in the warning?
```suggestion
            .addMember("message = %S", "This action has a newer major version: ${coords.deprecatedByVersion}")
```

Reply:
Done - I used the name of the class instead of just the version